pandas/tests/series/test_analytics.py

```diff
@@ -1255,6 +1255,17 @@ def test_isin_empty(self, empty):
         result = s.isin(empty)
         tm.assert_series_equal(expected, result)
 
+    def test_isin_cats(self):
```

Comment:
This can go in `pandas/tests/category/test_algos.py`.

Reply:
OK.